delivery/handlers/organise_handlers.py

```diff
@@ -81,3 +85,108 @@ def post(self, runfolder_id):
         except FileNameParsingException as e:
             log.error(str(e), exc_info=e)
             self.set_status(INTERNAL_SERVER_ERROR, reason=str(e))
+
+
+class OrganiseProjectAnalysisHandler(BaseOrganiseHandler):
+    """
+    Handler class for organizing a project after analysis in preparation for
+    staging and delivery.
+    """
+
+    def initialize(self, **kwargs):
+        self.config = kwargs["config"]
+        self.organise_service = kwargs["organise_service"]
+
+    @exception_handler
+    def post(self, analysis_pipeline, project):
+        organise_config_dir = pathlib.Path(self.config["organise_config_dir"])
+        organise_config_path = organise_config_dir / f"{analysis_pipeline}.md"
+
+        project_path = pathlib.Path(self.config["general_project_directory"]) / project
+
+        # TODO once we have not found exceptions, use these here
+        if not organise_config_path.is_file():
+            raise FileNotFoundError(
+                f"Config file not found at {organise_config_path}")
+        with open(organise_config_path, 'r') as organise_config_file:
+            config = yaml.load(organise_config_file, Loader=yaml.CLoader)
+        with open(organise_config_dir / "schema.json", 'r') as organise_config_schema:
+            schema = json.load(organise_config_schema)
+        jsonschema.validate(config, schema)
```

Comment:
for re-usability, it could be nice to break out the validation code to its own (static) function, perhaps even move it to another service if that fits with the overall code structure

maybe it's up for consideration if the validation should happen here in the handler or downstream in the service?

Reply:
also, this code is more or less repeated (with slight variations) in all three organize handlers, could you consider breaking it out to a helper function that can be called instead?